src/pudl/output/ferc1.py

```diff
@@ -59,7 +59,7 @@ def plants_steam_ferc1(pudl_engine):
         .assign(
             capacity_factor=lambda x: x.net_generation_mwh / (8760 * x.capacity_mw),
             opex_fuel_per_mwh=lambda x: x.opex_fuel / x.net_generation_mwh,
-            opex_nonfuel=lambda x: x.opex_production_total - x.opex_fuel,
+            opex_nonfuel=lambda x: x.opex_production_total - x.opex_fuel.fillna(0),
```

Comment:
are there instances in which we think the fuel cost could actually be non-zero and the `opex_production_total` does actually include it even though `opex_fuel` is null?

It might be nice to do a little checking of this `opex_production_total` before we make this assumption.

Reply:
fair

Reply:
Ok, for the steam table only 0.12% of the `opex_production_total` values don't add up to the sum of all other `opex_CATEGORY` columns. Most of these look like errors where the `opex_production_total` value is very low, in some cases single digits when it obviously shouldn't be.....this is probably something we should flag or fix down the road....but anyways these 36 bad values should probably **not** use the `production_total` value and rather the actual sum total (calculated). All the bad values' `production_total` values are _lower_ than the sum total of `opex_` columns so it's unlikely that they would include additional, "hidden" fuel values where `opex_fuel` is null. (Also there are only 4 bad rows where `opex_fuel` is null)

Given how exactly most of the opex values sum to equal the `opex_production_total` value regardless of whether there is a value for `opex_fuel`, I think that it's unlikely/other values would have to be reported wrong in order for `opex_fuel` to be null but a non-zero fuel amount was also included in the `production_total` value.

Reply:
Aren't we supposed to remove calculable columns in one of the ETL steps?

Reply:
HOWEVER -- for the small generators tables it DOES look like the fuel cost is NA but the total value INCLUDES FUEL.

Reply:
![image](https://user-images.githubusercontent.com/49878195/172264036-b988e30d-fd49-474c-8497-f81843e0e5cd.png)

Reply:
like, what is this^^^

Reply:
this is great news! thanks for investigating! that # of records are definitely ignore-able imho!